### PR TITLE
Update zulrah plugin overlay

### PR DESCRIPTION
**Pull request by devinfrench**
Preview

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/zulrah/ZulrahPlugin.java [hunk 1]
```diff
@@ -92,9 +108,11 @@ ZulrahConfig getConfig(ConfigManager configManager)
 	}
 
 	@Override
-	public Overlay getOverlay()
+	public Collection<Overlay> getOverlays()
 	{
-		return overlay;
+		Collection<Overlay> overlays = new ArrayList<>();
+		Collections.addAll(overlays, overlay, currentPhaseOverlay, nextPhaseOverlay, zulrahPrayerOverlay);
+		return overlays;
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/zulrah/ZulrahPlugin.java]:
```java
Arrays.asList(overlay, currentPhaseOverlay, nextPhaseOverlay, zulrahPrayerOverlay);
```

File: runelite-client/src/main/java/net/runelite/client/plugins/zulrah/overlays/ZulrahCurrentPhaseOverlay.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	@Inject
+	ZulrahCurrentPhaseOverlay(ZulrahPlugin plugin)
+	{
+		setPosition(OverlayPosition.BOTTOM_RIGHT);
+		setPriority(OverlayPriority.HIGH);
+		this.plugin = plugin;
+	}
+
+	@Override
+	public Dimension render(Graphics2D graphics, java.awt.Point parent)
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/zulrah/overlays/ZulrahCurrentPhaseOverlay.java]: Any reason for not importing the java.awt.Point directly here? At some places it conflicts with the Runelite API Point, but not here.

File: runelite-client/src/main/java/net/runelite/client/plugins/zulrah/overlays/ZulrahNextPhaseOverlay.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+	@Inject
+	ZulrahNextPhaseOverlay(ZulrahPlugin plugin)
+	{
+		setPosition(OverlayPosition.BOTTOM_RIGHT);
+		setPriority(OverlayPriority.HIGH);
+		this.plugin = plugin;
+	}
+
+	@Override
+	public Dimension render(Graphics2D graphics, java.awt.Point parent)
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/zulrah/overlays/ZulrahNextPhaseOverlay.java]: Check my previous comment

File: runelite-client/src/main/java/net/runelite/client/plugins/zulrah/overlays/ZulrahPrayerOverlay.java [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+	ZulrahPrayerOverlay(@Nullable Client client, ZulrahPlugin plugin)
+	{
+		setPosition(OverlayPosition.BOTTOM_RIGHT);
+		setPriority(OverlayPriority.MED);
+		this.client = client;
+		this.plugin = plugin;
+	}
+
+	@Override
+	public Dimension render(Graphics2D graphics, java.awt.Point parent)
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/zulrah/overlays/ZulrahPrayerOverlay.java]: check my previous comment

**Pull request merged**